### DISCUSSION TITLE
[Nextra 3] wrap the MDX content in the layout not working

### DISCUSSION THREAD

**Discussion by ycycxz** [shuding/nextra#2484] [Q&A]
Attempting to migrate to Nextra 3, the following customized components are not working as expected. Markdown content is not displaying in page/main.

1. To share a layout around MDX pages, create a layout component:`/components/layout.tsx`

```tsx
import Header from "./header";
import type { ReactNode } from "react";

export default function Layout({ children }: { children: ReactNode }) {

  return (
    <>
      <Header />
      <main>{children}</main>
    </>
  );
}
```

2. Then, import the layout component into the MDX page, wrap the MDX content in the layout, and export it: `/pages/main.mdx`

```mdx
import Layout from "../components/layout";
import { Steps, Tab, Tabs } from 'nextra/components'

export default function IndexPage({ children }) {
  return (
    <Layout>
    {children}
    </Layout>
  );
}

## title

Paragraph
text 
```

**Comment by dimaMachina** [comment]
Provide reproduction, migration guide, blog post coming
